client/app/components/Map/Map.jsx

```diff
@@ -1,22 +1,28 @@
 import React from "react";
-import store from '../../STORE.jsx';
-import ReactDOM from 'react-dom';
+import { store } from '../../STORE.jsx';
```

Comment:
we have to pass store implicitly through the parent app component if we do not import it here. a future PR will make this change for all child components of the app if we so choose.